### ISSUE TITLE
Save only a part of the model in Probe, load only the part in MockModel, and convert it to a tensor representing the whole model.

### ISSUE THREAD

**Issue by qualiteger** [qualiteg/ChatStream#35]

**Comment by qualiteger** [comment]
最後の logitのみ保存するには以下のようにすればよい

```
        relevant_logits = logits[0][-1]
        torch.save(relevant_logits, file_path)
```

これにより [50000] の形状のテンソルが保存される。

一方、読み込み側(MockModel)では、

loaded_logits = torch.load(file_path) で読み出したテンソルは [50000] なので chat_core側で  last_token_logits = logits[0][-1] 
とアクセスされてしまうと具合が悪い

そこで、 形状を [1, 1, 語彙サイズ] に変更し last_token_logits = logits[0][-1]としてアクセスできるように

```
        logits = loaded_logits.unsqueeze(0).unsqueeze(0)
```

のように二重のunsqueeze をしてやる。
